Assignment/solution_student/index.html

```diff
@@ -16,15 +18,62 @@
     <div class="container">
     
       <header>
-        <h1>FEWD HTML Template</h1>
+        <h1>Relaxr</h1>
+        <ul>
```

Comment:
Just as a bonus I would wrap this `ul` in a `nav` element, since it is the site navigation